sagemaker_endpoints/a_b_testing/a_b_testing.ipynb

```diff
@@ -2,7 +2,17 @@
  "cells": [
```

Comment:
<p>Better to have this uncommented so the user needs to purposefully comment it out to keep the endpoint</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Done</p>